client/src/reducers/index.js

```diff
@@ -124,4 +131,106 @@ export default combineReducers({
                 return conversation;
         }
     },
+    businesses: (businesses = defaultBusinesses, action) => {
+        switch (action.type) {
+            case ADD_BUSINESS: {
+                const id = Math.random();
```

Comment:
This will set `id` to a random float number between `[0, 1]`. Seems like integers would be better suited for this purpose.

Reply:
This also doesn't take it into account that the produced value may have already been taken. This can be addressed later, however, by creating a new issue!